Artnet/Receiver.h

```diff
@@ -319,7 +319,7 @@ class Receiver_ {
     }
     template <typename T = S>
     auto macAddress(uint8_t* mac) -> std::enable_if_t<std::is_same<T, EthernetUDP>::value> {
-#ifndef ESP8266
+#if !defined(ESP8266) && !defined(ARTNET_ETHER_IS_ENC28J60)
         Ethernet.MACAddress(mac);
 #endif
```

Comment:
The latest (on GitHub) version has the `MACAddress()` method.

https://github.com/JAndrassy/EthernetENC/blob/3a50d3aa9c1510e4090d1763ec3e91c700b764a1/src/Ethernet.h#L99

So, could you modify these lines as follows? (`ESP8266` also has `MACAddress()` method now, `#if` directive has removed)

```suggestion
        Ethernet.MACAddress(mac);
```

Reply:
Hi,

I was able to compile my sketch after cloning the latest ENC  library and removing the `#if` directive

Readme and workflow have been updated to use git instead of regular installation